model/k8s/metrics.yaml

```diff
@@ -60,3 +82,25 @@ groups:
       Total memory usage of the Node
     instrument: gauge
     unit: "By"
+
+  # k8s.node.network.* metrics
+  - id: metric.k8s.node.network.io
+    type: metric
+    metric_name: k8s.node.network.io
+    stability: experimental
+    brief: "Node network IO"
+    instrument: counter
+    unit: "By"
+    attributes:
+      - ref: system.device
+      - ref: network.io.direction
+  - id: metric.k8s.node.network.errors
+    type: metric
+    metric_name: k8s.node.network.errors
+    stability: experimental
+    brief: "Node network errors"
+    instrument: counter
+    unit: "{error}"
+    attributes:
+      - ref: system.device
+      - ref: network.io.direction
```

Comment:
is there any information about the error we can include? We have a very vague `error.type` attribute intended for this purpose.

Reply:
I don't think we have any information about the errors' types. It seems these are aggregated errors over a specific interface: https://github.com/kubernetes/kubernetes/blob/v1.31.1/staging/src/k8s.io/kubelet/pkg/apis/stats/v1alpha1/types.go#L189-L204